api/contactsApi.ts

```diff
@@ -12,297 +12,299 @@ const contactCreateInput = [
   {
     key: "workspaceId",
     required: true,
-    label: "workspaceId",
-    helpText: "Test",
+    label: "Workspace ID",
+    helpText: "The identifier for the workspace.",
     dynamic: "workspaces.id.projectName",
   },
   {
     key: "firstName",
     required: true,
-    label: "FirstName",
-    helpText: "Test",
+    label: "First Name",
+    helpText: "The first name of the contact",
   },
   {
     key: "lastName",
     required: true,
-    label: "LastName",
-    helpText: "Test",
+    label: "Last Name",
+    helpText: "The last name of the contact.",
   },
   {
     key: "workMobilePhone1",
     required: true,
-    label: "WorkMobilePhone1",
-    helpText: "Test",
+    label: "Work Mobile Phone 1",
+    helpText: "The primary work mobile number of the contact.",
   },
   {
     key: "workEmailAddress1",
     required: true,
-    label: "WorkEmailAddress1",
-    helpText: "Test",
+    label: "Work Email Address 1",
+    helpText: "The primary work email address of the contact.",
   },
   {
     key: "workCountry",
     required: true,
-    label: "WorkCountry",
-    helpText: "Test",
+    label: "Work Country",
+    helpText: "The work country of the contact.",
   },
   {
     key: "timezone",
     required: true,
-    label: "Timezone",
-    helpText: "Test",
+    label: "Time Zone",
+    placeholder: "Australia/Melbourne",
+    helpText:
+      "The time zone in which the contact lives in. Values are relative to GMT",
```

Comment:
Can we update to  `"The timezone in which the contact lives. Values are relative to GMT, please use Timezone 
ID format e.g Australia/Melbourne",`